Resources/doc/getting_started/the_list_view.rst

```diff
@@ -0,0 +1,215 @@
+                ->add('draft')
+            ;
+        }
+    }
+
+When saving this, you can now see that the title field has the link you were
+looking for.
+
+Displaying Other Models
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Now you probably also want the Category to be included in the list. To do that,
+you need to reference it. You can't add the ``category`` field to the list
+mapper, as it will then try to show the entity as a string. As you've learned
+in the previous chapter, adding ``__toString`` to the entity is not recommended
+as well.
+
+Fortunately, there is an easy way to reference other models by using the dot
+notation. Using this notation, you can specify which fields you want to show.
+For instance, ``category.name`` will show the ``name`` property of the
+category.
+
+.. code-block:: php
+
+    // src/AppBundle/Admin/BlogPostAdmin.php
+    namespace AppBundle\Admin;
+
+    // ...
+    class BlogPostAdmin extends Admin
+    {
+        // ...
+
+        protected function configureListFields(ListMapper $listMapper)
+        {
+            $listMapper
+                ->addIdentifier('title')
+                ->add('category.name')
+                ->add('draft')
+            ;
+        }
+    }
+
+Adding Filter/Search Options
+----------------------------
+
+Assume you had a very succesfull blog site containing many blog posts. After a
+while, finding the blog post you wanted to edit would be like finding a needle
+in a haystack. As with all user experience problems, Sonata provides a solution
+for it!
+
+It does this by allowing you to configure datagrid filters in the
+``Admin#configureDatagridFilters()`` method. For instance, to allow the admin
+to search blog posts by title (and also order them by alfabet in the list), you
+would do something like:
+
+.. code-block:: php
+
+    // src/AppBundle/Admin/BlogPostAdmin.php
+    namespace AppBundle\Admin;
+
+    use Sonata\AdminBundle\Datagrid\DatagridMapper;
+
+    // ...
+    class BlogPostAdmin extends Admin
+    {
+        protected function configureDatagridFilters(DatagridMapper $datagridMapper)
+        {
+            $datagridMapper->add('title');
+        }
+    }
+
+This will add a little block to the left of the block showing a search input
+for the title field.
+
+Filtering by Category
+~~~~~~~~~~~~~~~~~~~~~
+
+Filtering by another model's properties is a little bit more difficult. The add
+field has 5 arguments:
+
+.. code-block:: php
+
+    public function add(
+        $name,
+
+        // filter
+        $type = null,
+        array $filterOptions = array(),
+
+        // field
+        $fieldType = null,
+        $fieldOptions = null
+    )
+
+As you can see, you can both customize the type used to filter and the type
+used to display the search field. You can rely on the type guessing mechanism
+of Sonata to pick the correct field types. However, you still need to configure
+the search field to use the ``name`` property of the Category:
+
```

Comment:
I've written this doc while creating an admin, so everything in here did work as documented. Not sure if it's working now